### PR TITLE
PROJECT LOUNGE - Adding a team page

### PR DESCRIPTION
**Pull request by jodiedoubleday**
## What does this PR do?
- Creates a team page for everyone at Holiday Extras who has written a blog post
- Uses a Gravatar for the persons photo, has a HX themed fallback if no Gravatar exists.
- Adds the users photo to their blog post

## How to test
- Checkout branch
- Run `jekyll serve` this should boot up the blog locally (if not try `bundle exec jekyll s`)
- Look at the team page and decided if it looks ok to release LIVE to the world
- Check a few blog posts for photos

### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this on mobile Android, Safari, Chrome for any visual changes.
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

#### Software Engineer, Software Architect or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

### CODE REVIEW

**Comment by lukefrake** [comment]
I'll take whatever I equal review on this

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 _site
+node_modules
```
Comment (lukefrake) [inline .gitignore]: should only need /

File: _includes/head.html [hunk 2]
```diff
@@ -42,6 +42,6 @@
     <meta name="twitter:creator:id" "">
     -->
 
-    {% include styles.html %}
+    <style>{% include styles.html %}</style>
```
Comment (lukefrake) [inline _includes/head.html]: Why are styles in an html file?

File: _includes/styles.html [hunk 3]
```diff
@@ -81,7 +80,7 @@
 }
 nav ul li a {
   font-size:1.4em;
-  padding:0.5em 1em;
+  padding:0.5em 0.5em;
```
Comment (lukefrake) [inline _includes/styles.html]: Can just be padding: .5em

File: team/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+title: The Holiday Extras Tech Team
+---
+<div class="team">
+  <header>
+    <div class="container">
+      <h1>The tech team</h1>
+      <p>at <span>Holiday Extras</span></p>
+    </div>
+  </header>
+  <main class="">
```
Comment (lukefrake) [inline team/index.html]: empty class

File: team/index.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    <ul>
+      {% for member_hash in site.data.members %}
+        {% assign member = member_hash[1] %}
+        <li>
+          {% if member.gravatar_email %}
```
Comment (lukefrake) [inline team/index.html]: Maybe line 17-21 can be moved into a partial

**Comment by lukefrake** [comment]
Team page for me loads up empty, not sure why though. Might be my set up.

File: _includes/head.html (hunk 2 above)
Comment (jodiedoubleday) [inline _includes/head.html]: was looking into critical.js to put critical styles in the `<head>`, it missed about 2 lines so ive just put all of it in the head... for now

File: team/index.html (hunk 5 above)
Comment (jodiedoubleday) [inline team/index.html]: Yup, this will be something we fix in the next project lounge

Comment (lukefrake) [inline team/index.html]: Awesome!

File: _includes/head.html (hunk 2 above)
Comment (lukefrake) [inline _includes/head.html]: Okay, so performance reasons & will be changed in the future. Sweet!

**Comment by andrewcashmore** [comment]
Looks good - http://grab.by/FfSS

**Comment by andrewcashmore** [comment]
:-1:

**Pull request merged**